src/main/lrsql/admin/interceptors/jwt.clj

```diff
@@ -23,13 +23,16 @@
                        admin-u/header->jwt)
             result (if no-val?
                      ;; decode jwt w/o validation and ensure account
-                     (let [{:keys [issuer username] :as result}
-                           (admin-u/proxy-jwt->username-and-issuer
-                            token no-val-uname no-val-issuer no-val-role-key
-                            no-val-role)]
-                       (if (some? username)
-                         (:result (adp/-ensure-account-oidc lrs username issuer))
-                         result))
+                     (try
+                       (let [{:keys [issuer username] :as result}
+                             (admin-u/proxy-jwt->username-and-issuer
+                              token no-val-uname no-val-issuer no-val-role-key
+                              no-val-role)]
+                         (if (some? username)
+                           (:result (adp/-ensure-account-oidc lrs username issuer))
+                           result))
+                       (catch Exception _
+                         :lrsql.admin/unauthorized-token-error))
```

Comment:
I think the Exception can be made more specific. Otherwise any old exception can bubble up and be interpreted as a 401, e.g. if there's something wrong with the SQL layer.

Reply:
Discussed in Slack - this is a special case to catch all exceptions as 401 since this is auth